Logstash/confd/templates/user-service.cloudwatch.conf.tmpl

```diff
@@ -11,38 +11,13 @@ filter {
   mutate{
       replace => [ "message", "%{message}" ]
       gsub => [ 'message','\n','']
+      gsub => [ 'message','\"\"([^,])','"\1'] # nginx logs were malformed with an extra " on the realip key
   }
 
-  if [message] =~ /^{.*}$/ {
-    json { 
-      skip_on_invalid_json => true 
-      source => message
-      }
-  }
-  else {
-    grok {
-        match => { "message" => ["%{IPORHOST:[nginx][access][remote_ip]} - %{DATA:[nginx][access][user_name]} \[%{HTTPDATE:[nginx][access][time]}\] \"%{WORD:[nginx][access][method]} %{DATA:[nginx][access][url]} HTTP/%{NUMBER:[nginx][access][http_version]}\" %{NUMBER:[nginx][access][response_code]} %{NUMBER:[nginx][access][body_sent][bytes]} \"%{DATA:[nginx][access][referrer]}\" \"%{DATA:[nginx][access][agent]}\""] }
```

Comment:
the json should be unfolded whatever is the object logs, so we don't need to specifically go through each field like this